catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -415,6 +436,7 @@ protected void onDestroy() {
 		CameraManager.getInstance().releaseCamera();
 		CameraManager.getInstance().setToDefaultCamera();
 		ProjectManager.getInstance().setSceneToPlay(ProjectManager.getInstance().getCurrentScene());
+		CastManager.getInstance().onStageDestroyed();
```

Comment:
if its not a cast project, an unnecessary castManager would be instantiated at this point....